mailserver/mailserver.go

```diff
@@ -157,20 +162,27 @@ func (s *WMailServer) Close() {
 }
 
 // Archive a whisper envelope.
```

Comment:
Would be nice to add an explanation to the document regarding the meaning of the new return value

Reply:
@PombeirP I rolled back that change. I was using the return key just in tests, but it doesn't make sense to patch `go-ethereum`for this reason I think.